CHANGES.md

```diff
@@ -17,6 +17,7 @@ Change Log
 * Fixed a bug that prevented `setView` from working across all scene modes.
 * Fixed a bug that caused `camera.positionWC` to occasionally return the incorrect value.
 * Used all the template urls defined in the CesiumTerrain provider.[#3038](https://github.com/AnalyticalGraphicsInc/cesium/pull/3038)
+* Add imageryLayer object as a property to ImageryLayerFeatureInfo.
```

Comment:
Sorry, I should have mentioned: put this in a new 1.15 section at the top, dated 2015-11-02.